cmd/admin-handlers.go

```diff
@@ -70,15 +70,31 @@ func (adminAPI adminAPIHandlers) ServiceStatusHandler(w http.ResponseWriter, r *
 		return
 	}
 
+	objLayer := newObjectLayerFn()
+	if objLayer == nil {
+		writeErrorResponse(w, ErrServerNotInitialized, r.URL)
+		return
+	}
+
 	// Fetch storage backend information
-	storageInfo := newObjectLayerFn().StorageInfo()
+	storageInfo := objLayer.StorageInfo()
```

Comment:
@harshavardhana what does ``status`` field in the above json stand for? IIUC, ``service`` could be ``online`` or ``offline``, possibly ``maintenance`` in the future (for servers with all disks unformatted in a distributed setup). Is that right?